server/tests/scripts/run_testbed_server.py

```diff
@@ -169,7 +169,8 @@ async def test_drop(organization_id: str):  # type: ignore[misc]
             print(
                 f"{BOLD_YELLOW}All set !"
                 f"{NO_COLOR} Don't forget to export `TESTBED_SERVER_URL` environ variable:\n"
-                f"{PINK}export TESTBED_SERVER_URL='parsec://127.0.0.1:{port}?no_ssl=true'{NO_COLOR}"
+                f"{PINK}export TESTBED_SERVER_URL='parsec://127.0.0.1:{port}?no_ssl=true'{NO_COLOR}",
+                flush=True,
```

Comment:
🫣 That's why we don't have output last time :sweat:

Reply:
Yes, @vxgmichel found it